src/cljs/rems/table.cljs

```diff
@@ -68,14 +67,18 @@
     m
     (assoc m k (make-default m))))
 
-(defn- apply-row-defaults [row]
+(defn apply-row-defaults [row]
```

Comment:
I had to change this function to public to test it outside the namespace -- otherwise `lein doo once` gives the warning `WARNING: var: rems.table/apply-row-defaults is not public at line 12 test/cljs/rems/test_table.cljs`.

Are there any better solutions?

Reply:
The usual trick is to refer to privates like `@#'rems.table/apply-row-defaults`, but I'm not sure if that works in cljs.
Another option is to have the test in this file.
Making the function public is fine.